package.json

```diff
@@ -31,12 +31,15 @@
     "check": "tsc",
     "prepublishOnly": "pnpm test && dts-buddy",
     "sandbox": "node test/sandbox/index.js",
-    "test": "vitest --run"
+    "test": "vitest --run",
+    "test:ui": "vitest --ui"
   },
   "license": "MIT",
   "dependencies": {
     "@jridgewell/sourcemap-codec": "^1.4.15",
-    "@types/estree": "^1.0.1"
+    "@typescript-eslint/types": "^8.2.0",
+    "@vitest/ui": "^2.0.5",
+    "vitest": "^2.0.5"
```

Comment:
looks like `vitest` was accidentally moved to `dependencies` instead of `devDependencies`